popup/popup.css

```diff
@@ -25,8 +43,20 @@ body, html {
 }
 
 #folderList {
+  --folder-list-search-bgcolor: var(--menu-background-color-search);
+  --folder-list-search-color: var(--menu-color-search);
+
+  --folder-list-item-color: var(--menu-color);
+
   --folder-list-item-bgcolor-hover: var(--menu-background-color-select);
   --folder-list-item-color-hover: var(--menu-color-select);
+  /*--folder-list-item-border*/
```

Comment:
Did you want to make this one a rule too? If this is for future changes I'd prefer adding it in a different PR.

Reply:
I do not remember why I put this in. I guess I copied it from some other example.

Reply:
```suggestion
```